ground/src/main/java/com/google/android/ground/persistence/sync/SurveySyncWorker.kt

```diff
@@ -53,9 +54,8 @@ constructor(
       Timber.d("Syncing survey $surveyId")
       syncSurvey(surveyId)
     } catch (t: Throwable) {
-      val logger = FirebaseCrashLogger()
-      logger.setSelectedSurveyId(surveyId)
-      logger.logException(t)
+      firebaseCrashLogger.setSelectedSurveyId(surveyId)
+      Timber.e(t)
```

Comment:
Let's also log text explaining what may have caused this.

Reply:
You can disregard this, I added these changes to my PR.

Reply:
Ack